test/helpers/multisig.js

```diff
@@ -0,0 +1,48 @@
+const deployOrGetRootRegistry = require('./contracts').deployOrGetRootRegistry;
+
+const prepareMultiSigAndRoot = async config => {
+  const { network, artifacts, accounts } = config;
+  const admin0 = accounts[0];
+  const admin1 = accounts[1];
+  let multiAdmin, multiSigWallet;
+  const rootRegistry = await deployOrGetRootRegistry(config);
+  // on ropsten we always want to get the multisigs from the root, so only in development do we want to create new ones
+  if (network === 'develop' || network === 'test') {
+    [multiSigWallet, multiAdmin] = await Promise.all([
+      artifacts
+        .require('MultiSigWallet')
+        .new([admin0, admin1], 1, rootRegistry.address),
+      artifacts
+        .require('MultiAdmin')
+        .new([admin0, admin1], 1, rootRegistry.address),
+    ]);
+    await rootRegistry.setVersion(
+      'MultiAdmin',
+      multiAdmin.address,
+      '0_1_0',
+      multiAdmin.address
+    );
+    await rootRegistry.setVersion(
+      'MultiSigWallet',
+      multiSigWallet.address,
+      '0_1_0',
+      multiSigWallet.address
+    );
```

Comment:
Can you add a README.md file somewhere that explains which contracts are in which registry?

Reply:
sure, creating a ticket